xcpretty.gemspec

```diff
@@ -26,6 +26,8 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_dependency 'rouge', '~> 1.8'
+
```

Comment:
> Due to 336e394, I don't think this is a good idea and I think we should close this PR.

I'm sorry if I caused a conversation, that wasn't the intention.
Thank you again for your work - it's a great thing and we should merge this PR.